<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLDisplayNodeWidget</class>
 <widget class="QWidget" name="qMRMLDisplayNodeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display Node</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox">
     <property name="title">
      <string>Visibility</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="VisibilityLabel">
        <property name="text">
         <string>&amp;Visibility:</string>
        </property>
        <property name="buddy">
         <cstring>VisibilityCheckBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="VisibilityCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="SelectedLabel">
        <property name="text">
         <string>&amp;Selected:</string>
        </property>
        <property name="buddy">
         <cstring>SelectedCheckBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="SelectedCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="ClippingLabel">
        <property name="text">
         <string>&amp;Clip:</string>
        </property>
        <property name="buddy">
         <cstring>ClippingCheckBox</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="ClippingCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="SliceIntersectionVisibilityLabel">
        <property name="text">
         <string>Slice &amp;Intersections Visibility:</string>
        </property>
        <property name="buddy">
         <cstring>SliceIntersectionVisibilityCheckBox</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="SliceIntersectionVisibilityCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="SliceIntersectionThicknessLabel">
        <property name="text">
         <string>Slice Intersections &amp;Thickness:</string>
        </property>
        <property name="buddy">
         <cstring>SliceIntersectionThicknessSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="SliceIntersectionThicknessSpinBox">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="SliceIntersectionOpacityLabel">
        <property name="text">
         <string>Slice Intersections Opacity:</string>
        </property>
        <property name="buddy">
         <cstring>SliceIntersectionOpacitySlider</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="ctkSliderWidget" name="SliceIntersectionOpacitySlider">
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
        <property name="SingleStep" stdset="0">
         <double>0.010000000000000</double>
        </property>
        <property name="Decimals" stdset="0">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLDisplayNodeViewComboBox" name="DisplayNodeViewComboBox">
        <property name="toolTip">
         <string>Select views in which to show this node. All unchecked shows in all 3D and 2D views.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="DisplayNodeViewLabel">
        <property name="text">
         <string>View:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="ThreeDVisibilityLabel">
        <property name="text">
         <string>3D Visibility:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="ThreeDVisibilityCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="ctkVTKPropertyWidget" name="PropertyWidget"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLCheckableNodeComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLCheckableNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLDisplayNodeViewComboBox</class>
   <extends>qMRMLCheckableNodeComboBox</extends>
   <header>qMRMLDisplayNodeViewComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkVTKPropertyWidget</class>
   <extends>QWidget</extends>
   <header>ctkVTKPropertyWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
